include/antler/system/utils.hpp

```diff
@@ -72,8 +72,24 @@ namespace antler::system {
       std::size_t n;
 
       while ((n = fread(buff.data(), 1, buff.size(), h)) > 0) {
-         fwrite(buff.data(), 1, n, stdout);
+         fwrite(buff.data(), n, 1, stdout);
       }
+
+      return WEXITSTATUS(pclose(h));
+   }
+
+   inline static int32_t execute_quiet(std::string cmd, const std::vector<std::string>& args) {
+      std::string_view prog = cmd;

```

Comment:
Same defect as above.
```suggestion
   inline static int32_t execute_quiet(std::string_view prog, const std::vector<std::string>& args) {
      std::string cmd{prog};
```

Reply:
fixed in 17da588